### DISCUSSION TITLE
Linking to Gentoo docs for LTO

### DISCUSSION THREAD

**Discussion by immolo** [InBetweenNames/gentooLTO#880] [General]
Any chance of dropping a URL so users know there is now a choice of using LTO without needing an overlay?

There is general confusion currently for new users as Gentoo isn't ready to say its safe to use LTO in the handbook for everyone yet however, with some more testing this will day will come a lot sooner.

https://wiki.gentoo.org/wiki/LTO is the current wiki page for all Gentoo LTO information.

**Comment by powerpc64** [comment]
absolutely _no_ idea what this means but i support it :P

**Comment by mitchweaver** [comment]
Sure, you can add -flto without this repository. Have fun with half the packages not compiling.  
This project has workarounds to get most packages that fail to actually work with LTO, where in the wiki you linked Gentoo's official response is to just build them with clean CFLAGS ie no LTO

**Reply by immolo** [reply]
> Sure, you can add -flto without this repository. Have fun with half the packages not compiling. This project has workarounds to get most packages that fail to actually work with LTO, where in the wiki you linked Gentoo's official response is to just build them with clean CFLAGS ie no LTO

Not sure what you mean here? On my 10 Gentoo machines using LTO and with multiple architectures,  I don't use a single package env to disable LTO for nearly a year now. The wiki article also say to report the issue so it get fixed then use the no nolto method as a workaround which is different to this repo how?
